hdk/cl/developer_designs/cl_firesim/build/scripts/create_dcp_from_cl.tcl

```diff
@@ -143,10 +149,13 @@ set_msg_config -id {Constraints 18-619}  -suppress
 set_msg_config -id {DRC CKLD-2}          -suppress
 set_msg_config -id {DRC REQP-1853}       -suppress
 set_msg_config -id {Timing 38-436}       -suppress
+# https://forums.aws.amazon.com/forum.jspa?forumID=243&start=0
+set_msg_config -severity "CRITICAL WARNING" -string "WRAPPER_INST/SH" -suppress
+set_msg_config -severity "WARNING"          -string "WRAPPER_INST/SH" -suppress
 
 # Promote the following critical warnings to errors to prevent AGFI generation
 # Design not completely routed
-set_msg_config -id {Route 35-1} -new_severity "ERROR"
+#set_msg_config -id {Route 35-1} -new_severity "ERROR"
```

Comment:
Router Error Workaround

Reply:
Did the final answer from AWS come to you as a direct message?  Could you update https://forums.aws.amazon.com/thread.jspa?threadID=338653&tstart=0 and paste a link to that here?  Have you implemented the TCL checks for design not completely routed in this PR so that we won't hit it or are you going to do that in a follow on?